frontend/src/App.vue

```diff
@@ -83,10 +103,17 @@ export default {
         };
       }
     },
+    changeItemsState(changedItems) {
```

Comment:
I suggest choosing names that better portray the logical flow.
The flow is:

1. We get **new data contents**.
2. We locate the **instances** that should be modified
3. We overwrite the data on the **instances** with the new **changed data**

Does this description help in improving the variable names?